README

```diff
@@ -414,6 +414,15 @@ INSTALLATION
 
     By default INSTALLDIRS is set to site.
 
+DOCKER USAGE
```

Comment:
That's good for now. I'll update it with official image once automated build is setup.